scheduler/plugin/pluginextender.go

```diff
@@ -0,0 +1,60 @@
+package plugin
+
+import (
+	"context"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/scheduler/framework"
+)
+
+type PluginExtender interface {
```

Comment:
> scheduler/plugin/pluginextender.go:10:6: exported: type name will be used as plugin.PluginExtender by other packages, and that stutters; consider calling this Extender (revive)

This line gets the lint error. But I think `Extender` is also not good. Hard to understand what this interface is.